cdap-api/src/main/java/co/cask/cdap/api/flow/Flow.java

```diff
@@ -36,7 +36,9 @@
   /**
    * Configure the {@link Flow} by returning a {@link FlowSpecification}.
    *
-   * @return An instance of {@link FlowSpecification}.
+   * @return An instance of {@link FlowSpecification}
+   * @deprecated Use {@link AbstractFlow#configureFlow} instead
    */
+  @Deprecated
```

Comment:
Yes. in the javadoc, we point users to AbstractFlow#configureFlow method. We will probably rename configureFlow to configure and move it to Flow after we remove the deprecated method.